torsions/TorsionScanSet.py

```diff
@@ -203,7 +203,7 @@ def compute_energy(self, param, platform=None):
             energy = state.getPotentialEnergy()._value
             mm_energy.append(energy)
             #print "frame %5d / %5d : %8.3f kJ/mol" % (i, self.n_frames, energy) # DEBUG
-            delta_energy.append(self.qm_energy[i]._value - energy)
+            delta_energy.append(self.qm_energy[i]._value - energy + offset)
```

Comment:
I don't think it matters if the offset is added or subtracted. If I'm adding a negative value (which is the range I gave it) then I'm really subtracting. 

About the prior the offset - would a very broad normal be better? Since we're not sure what the limits of the offset should be?